al_aws_collector.js

```diff
@@ -126,6 +126,7 @@ class AlAwsCollector {
         this._customStatsFuns = statsFuns;
         this._collectorId = process.env.collector_id;
         this._stackName = process.env.stack_name;
+        this._applicationId = process.env.al_application_id;
```

Comment:
Does this mean should take this value out of paws collectors?

Reply:
Yes, I'll send a follow up PR.